apkid/rules/apk/packers.yara

```diff
@@ -604,3 +604,23 @@ rule secenh : packer
     and 1 of ($a*)
     and 1 of ($b*)
 }
+
+// There can also be rules for dex files because of NRV compression but I can't find compressed classesN.dex
+// I guess it is optional(?).
+rule legu : packer
+{
+  meta:
+    description = "Legu Packer"
+    sample = "9ff3a53f76c7a6d7e3de3b8567c9606f2cc08ec4aaaae596a27361018d839c58"
+    author = "Mert Arıkan"
+  strings:
+    $a = "assets/toversion"
```

Comment:
Is this file helping you to know the versions interval of this class of Tencent packer?

Reply:
Nop, It's just used for decryption, but you can use the name of the native library to identify the version

Reply:
LGTM, would be awesome to make these sort of comment, in the rule, using the yara comment syntax. This can help for maintaining the rule in the future without having to reference the github issues.